lib/Epub/Epub/blocks/TextBlock.cpp

```diff
@@ -46,6 +53,13 @@ std::unique_ptr<TextBlock> TextBlock::deserialize(File& file) {
 
   // words
   serialization::readPod(file, wc);
+
+  // Sanity check: prevent allocation of unreasonably large lists (max 10000 words per block)
+  if (wc > 10000) {
+    Serial.printf("[%lu] [TXB] Deserialization failed: word count %u exceeds maximum\n", millis(), wc);
+    return nullptr;
+  }
```

Comment:
Fair sanity check, this shouldn't be possible with https://github.com/daveallie/crosspoint-reader/blob/2771579007656f78fec756987679abb98cc1ded8/lib/Epub/Epub/parsers/ChapterHtmlSlimParser.cpp#L148-L157, but happy to leave in.